lib/rack/tracer.rb

```diff
@@ -11,14 +11,16 @@ class Tracer
     # @param tracer [OpenTracing::Tracer] A tracer to be used when start_span, and extract
     #        is called.
     # @param on_start_span [Proc, nil] A callback evaluated after a new span is created.
+    # @param on_span_finish [Proc, nil] A callback evaluated after a span is finished.
     # @param errors [Array<Class>] An array of error classes to be captured by the tracer
     #        as errors. Errors are **not** muted by the middleware, they're re-raised afterwards.
-    def initialize(app, tracer: OpenTracing.global_tracer, on_start_span: nil, trust_incoming_span: true, errors: [StandardError])
+    def initialize(app, tracer: nil, on_start_span: nil, on_span_finish: nil, trust_incoming_span: true, errors: nil)
```

Comment:
why `errors` change?

Reply:
and `tracer` as well. Lets please keep them as they were, as they are not related to this PR

Reply:
Just moved default options into the method body to make the method definition shorter and easier to read